pkg/plugins/resources/toml/spec.go

```diff
@@ -20,8 +20,11 @@ type Spec struct {
 	Value string `yaml:",omitempty"`
 	// [c][t] *Deprecated* Please look at query parameter to achieve similar objective
 	Multiple bool `yaml:",omitempty" jsonschema:"-"`
-	// [s]VersionFilter provides parameters to specify version pattern and its type like regex, semver, or just latest.
+	// [s] VersionFilter provides parameters to specify version pattern and its type like regex, semver, or just latest.
 	VersionFilter version.Filter `yaml:",omitempty"`
+	// [t] AllowsMissingKey allows non-existing keys. If the key does not exist, the key is created if AllowsMissingKey is true, otherwise an error is raised (the default)
+	// Only supported if Key is used
```

Comment:
In the case of a multi line comment,  we need to use this syntax

```
/*
   This is a comment
*/
```
The reason is the library generating the jsonschema uses the comment prior the struct field so in this case will only uses 

```
Only supported if Key is used
```

Reply:
Probably better to do this from the IDE so the formatting is correct

```suggestion
	/* 
	  [t] AllowsMissingKey allows non-existing keys. If the key does not exist, the key is created if            AllowsMissingKey is true, otherwise an error is raised (the default)
	  Only supported if Key is used
	*/ 
```